app/lib/activitypub/activity/delete.rb

```diff
@@ -21,11 +21,13 @@ def delete_person
   def delete_note
     return if object_uri.nil?
 
+    RedisLock.acquire(lock_options) do |lock|
+      delete_later!(object_uri)
```

Comment:
CodeClimate complains about us not using `lock`, but we can't leave it out.
We don't use it because we want to call `delete_later!` even in the unlikely case where we couldn't acquire the lock, it's not really an issue.

Reply:
We can totally leave it out though? Or just name it `_lock`

Reply:
We can't leave it out because Redislock checks for that parameter (see https://circleci.com/gh/tootsuite/mastodon/39022?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). I will name it `_lock`.